### PR TITLE
BOOT-142 Restore default bulma button padding, add bulma input class

### PR DESCRIPTION
**Pull request by WilliamHoangMMT**
https://bootbag.atlassian.net/browse/BOOT-142

Restored the default bulma padding on their `.button` class and added bulma's `.input` class to our email input field which adds rounded corners.

Had to add min-width to the form to ensure elements stayed centered since the new padding isn't responsive to changes in parent container size and also reduced the gap between input and button.

https://user-images.githubusercontent.com/129167065/229843301-04940d63-5956-4f0a-9349-ced20662eeb3.mov

### CODE REVIEW

File: layouts/default.vue [hunk 1]
```diff
@@ -461,7 +460,12 @@ form {
 
 @media only screen and (min-width: 768px) {
   form {
+    min-width: 500px;
```
Comment (DaisyyWard) [inline layouts/default.vue:463]: Would a percentage value be better here like 80%. I don't normally like to use fixed widths on these types of things

Comment (WilliamHoangMMT) [reply layouts/default.vue:463]: We're already using a % value on the line below (50%), there's just some edge case where after a certain point the container holding the input field and button gets smaller than the input field and button themselves causing the elements to not be centered at certain browser widths which is why I've added this fixed min-width

Comment (DaisyyWard) [reply layouts/default.vue:463]: Can you show me what it looks like without this min width value? At the minute I am wondering why this approach is better than a centering approach like flex?

Comment (WilliamHoangMMT) [reply layouts/default.vue:463]: Screenshot 2023-04-04 at 16 43 38

**Review by DaisyyWard — APPROVED** [review]

**Review by pravienmangalMMT — APPROVED** [review]

**Pull request merged**